src/data_model/transformations.jl

```diff
@@ -472,3 +472,917 @@ function apply_voltage_angle_difference_bounds!(eng::Dict{String,<:Any}, vad::Re
+                        if ismissing(v_start[(f_bus,t)])
+                            v_start[(f_bus,t)] = v_fr_prop[i]
+                            push!(stack, (f_bus,t))
+                        end
+                    end
+                end
+            elseif tr["configuration"]==DELTA
+                v_fr = Array{Union{Complex,Missing}}([v_start[(f_bus, t)] for t in f_conns])
+                v_to = Array{Union{Complex,Missing}}([v_start[(t_bus, t)] for t in t_conns])
+                # forward propagation
+                if all((!).(ismissing.(v_fr))) && any(ismissing.(v_to))
+                    if ismissing(v_to[end])
+                        v_to[end] = 0.0*im
+                    end
+                    v_to_n = v_to[end]
+                    M = _get_delta_transformation_matrix(length(v_fr))
+                    v_to = [(M*v_fr)./scale.+v_to_n..., v_to_n]
+                    for (i,t) in enumerate(t_conns)
+                        if ismissing(v_start[(t_bus,t)])
+                            v_start[(t_bus,t)] = v_to[i]
+                            push!(stack, (t_bus,t))
+                        end
+                    end
+                end
+                # backward propagation
+                if all((!).(ismissing.(v_to))) && any(ismissing.(v_fr))
+                    v_to_p = v_to[1:end-1]; v_to_n = v_to[end]
+                    M = _get_delta_transformation_matrix(length(v_fr))
+                    Mp = [M[1:end-1,:]; fill(1.0, 1, length(v_fr))]
+                    v_to_pn_scaled = (v_to_p.-v_to_n).*scale
+                    v_fr = inv(Mp)*[v_to_pn_scaled[1:end-1]..., 0.0]
+                    for (i,t) in enumerate(f_conns)
+                        if ismissing(v_start[(f_bus,t)])
+                            v_start[(f_bus,t)] = v_fr[i]
+                            push!(stack, (f_bus,t))
+                        end
+                    end
+                end
+            end
+        end
+        if verbose
+            progress = round(sum(ismissing.(values(v_start)))/length(v_start)*100, digits=2)
+            println("it. $count:\t$progress% left to initialize at end")
+        end
+    end
+
+    # increment non-grounded zero values with epsilon
+    for (k,v) in v_start
+        (b,t) = k
+        data_bus = data_math["bus"][string(b)]
+        grounded_t = Dict(data_bus["terminals"].=>data_bus["grounded"])
+        if !ismissing(v) && iszero(v) && !grounded_t[t]
+            v_start[k] = convert(Complex, epsilon)
+        end
+    end
+
+    return v_start
+end
+
+
+"""
+Adds start values for the voltage to the buses.
+For a multinetwork data model, you can calculate the start voltages for a representative network through 'calc_start_voltage',
+and pass the result as 'uniform_v_start' to use the same values for all networks and avoid recalculating it for each network. 
+The argument 'epsilon' controls the offset added to ungrounded terminals which would otherwise be set to zero.
+"""
```

Comment:
For exported functions, we should follow the new docstring pattern where the function is given (indented) first, see, e.g. transform_data_model function as an example.